scripts/release.sh

```diff
@@ -13,7 +13,7 @@ fi
 DIR=$(dirname "$0")
 cd "$DIR/.."
 
-./scripts/upload-assets.py latest
+./scripts/upload-assets.py latest --force
```

Comment:
I'm... honestly not sure if we want this or if we want to allow users to pipe the `--force` arg thru via `make release` / have a separate invocation `upload-assets.py latest --clean` that JUST deletes the existing bucket and then user can run `make release` without the `--force` flag. thoughts?

Reply:
I can come up with some pros and cons but they seem minor enough and the impact of this decision small enough that it's probably not worth our time.

Reply:
Actually, I'm in favor of the `--clean` option. As Nick has been saying, we should move the release process to CI, and the `--clean` option allows us to handle that error separately on our laptops.

Reply:
done!